.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: "Release"
+on:
+  push:
+    branches:
+      - master
+      - alpha
+
+jobs:
+  # https://docs.github.com/en/actions/guides/publishing-nodejs-packages#publishing-packages-using-yarn
+  # https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/github-actions.md
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+
+      - run: yarn install
+
+      - run: yarn run build
+
+      - run: npx semantic-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@DiederikvandenB can you please get this from npm and add it through the repository settings?